pages_list.py

```
@@ -1,50 +1,57 @@
-import re
-
-from selenium.webdriver.firefox.service import Service
-
-import config
-import logging
-import database
+##########################################################################
+###################### First Latvian Fasker' Ripper ######################
+#################### Copyright (c) 2024-2025 mr.Iceman ###################
+##########################################################################
 from page import *
```

Ruff (0.8.2)

`from page import *` used; unable to detect undefined names

(F403)